integration-tests/src/test_loop/utils/setups.rs

```diff
@@ -0,0 +1,54 @@
+//! This file contains standard setups for test loop tests.
+//! Using TestLoopBuilder gives a lot of flexibility, but sometimes you just need some basic blockchain.
+
+use itertools::Itertools;
+use near_chain_configs::test_genesis::TestGenesisBuilder;
+use near_primitives::types::AccountId;
+
+use crate::test_loop::builder::TestLoopBuilder;
+use crate::test_loop::env::TestLoopEnv;
+use crate::test_loop::utils::ONE_NEAR;
+
+/// 2 producers, 2 validators, 1 rpc node, 4 shards, 20 accounts (account{i}) with 10k NEAR each.
+pub fn standard_setup_1() -> TestLoopEnv {
```

Comment:
IMO, it doesn't make sense to over simplify the setup phase. It already feels concise enough to and standardizing it into a function without any parameters probably invites more complexity and layering, and hinders readability.

If you'd really like, we can add more functionality for different types of nodes in the testloop builder instead.

Reply:
90% of the tests I've written would work just fine on a basic, default blockchain with a few validators and an rpc node. I think it would be useful to have a simple way to create a basic blockchain that most tests can use.
It takes 40 lines of code to set up a testloop test, IMO that isn't concise.
Without a common function all tests will have their own 40 line setup function.

`TestEnv` was easier to set up, just a few lines of code, e.g:
```rust
let num_nodes = 4;
let validators = (0..num_nodes)
    .map(|i| AccountId::try_from(format!("test{}", i + 1)).unwrap())
    .collect::<Vec<_>>();
let mut env = TestEnv::new(vec![validators.clone()], 5, false);
```

Reply:
Apart from shorter code, having a standard test setup could make the tests more readable. Reading a test written by someone else is always easier when they use standard, well known conventions, account names, patterns, etc.

Reply:
Sure, let's go with this but we should make sure to advertise about this function to all new users. I'm still not super convinced but it's definitely out of scope for this PR :)